deployer/deployer-services/src/main/java/io/hyscale/deployer/services/util/K8sResourceDispatcher.java

```diff
@@ -19,6 +19,7 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
We can remove the import if its not used. Even in other files also.

Reply:
sure.